README.md

```diff
@@ -86,7 +88,9 @@ Run the server with
 
 ```
 
-This will generate a JWT for you to log into the application. The script prints out the URL with the token included.
+Note, you will also need to run an upstream tool (eg, https://github.com/ONSDigital/go-launch-a-survey) to launch a survey. 
+
```

Comment:
I think it would be useful to provide the docker command here to run the latest launcher